backend/app/create_tables.py

```
@@ -1,10 +1,11 @@
-
 from database import engine, Base
-import backend.user_management.models as models 
+import backend.app.models as models
```

Ruff (0.8.2)

`backend.app.models` imported but unused

Remove unused import: `backend.app.models`

(F401)